solidity/contracts/Chainlinked.sol

```diff
@@ -166,6 +166,11 @@ contract Chainlinked {
     return _run.requestId;
   }
 
+  function completeChainlinkFulfillment(bytes32 _requestId)
+    internal
+    checkChainlinkFulfillment(_requestId)
```

Comment:
I recently read that modifiers are more expensive than methods, still need to verify. Either way, I think it'd make more sense to have the modifier call the method instead of the other way around.

Reply:
Which scenario would be more common? As-is, we assume using the modifier would be more common, and thus less expensive for the majority of users. If we place the function within the modifier, it would only be cheaper for (assuming) fewer users.

Reply:
What I'd heard was that modifiers are in general more expensive than methods, if the check they perform is equal. We can verify this, but I still think it's strange to have an empty method that is only used to call a modifier. I think having the modifier call the method would generally be a bit more intuitive.